### ISSUE TITLE
the debugger starts with nothing happening

### ISSUE THREAD

**Issue by y1xiaoc** [DonJayamanne/javaVSCode#27]
hello, I've been trying to use VSCode to write and debug java. However after I set up the configuration and click the debug button, **nothing** happened. 

Here are pictures about the problem. The first one shows my configuration and the second one shows the state after I clicked the debug button. Please ignore the folder named "1".
<img width="960" alt="config" src="https://cloud.githubusercontent.com/assets/19890787/23409425/8abd523c-fe06-11e6-8a59-625e2968658c.png">
<img width="960" alt="debug" src="https://cloud.githubusercontent.com/assets/19890787/23409426/8abe2ee6-fe06-11e6-8f56-e33ca7c25adf.png">

Even if I change the startup class into a non-exist class the debugger still remains the same and throw no error. Any body can help me? Lots of thanks!

**Comment by devin-sxp** [comment]
I have the same question, please！how do you fix it?

**Comment by yagupta** [comment]
+1 same problem.

**Comment by faustinoaq** [comment]
Sometimes this happens, I just stop the debugger and run again.

Could be that java debug server is waiting for connection.

**Comment by TenviLi** [comment]
+1 same problem
beg someone!!!!!!!Could you help me!!!!??!?!?!??!

**Comment by faustinoaq** [comment]
@gylidian this eventually happens, just try to reload your editor and debug again.

mine is working as expected.

![screenshot_20170807_124414](https://user-images.githubusercontent.com/3067335/29038663-203c91e4-7b6e-11e7-8241-fc9ce065071b.png)

**Comment by TenviLi** [comment]
@faustinoaq 
I am just a junior middle school student;During these several days,I am always confused with this problem.
My vscode can only run,but cannot debug.
Follow your saying,after reload,nothing changed.

This is my launch.json
`{
    "version": "0.2.0",
    "configurations": [{
            "name": "Java",
            "type": "java",
            "request": "launch",
            "stopOnEntry": true,
            "jdkPath": "D:/Java/jdk1.8.0_144/bin",
            "cwd": "${fileDirname}",
            "startupClass": "${fileBasenameNoExtension}",
            "classpath": [
                ".",
                "${fileDirname}"
            ]
        },
        {
            "name": "Java Console App",
            "type": "java",
            "request": "launch",
            "stopOnEntry": true,
            "jdkPath": "${config:java.home}/bin",
            "cwd": "${fileDirname}",
            "startupClass": "${fileBasenameNoExtension}",
            "classpath": [
                ".",
                "${fileDirname}"
            ],
            "externalConsole": true
        }
    ]
}`

I want to cry

**Comment by faustinoaq** [comment]
Your `launch.json` looks good, I don't know what could be the problem :sweat_smile:

**Comment by TenviLi** [comment]
@faustinoaq 
Thank you all the same!  ~~😸

**Comment by hspeed** [comment]
your jdk language is not english

**Comment by huijian142857** [comment]
I have the same problem, don't know why

**Comment by faustinoaq** [comment]
Yeah, It is a [known issue](https://github.com/DonJayamanne/javaVSCode#known-issues).

I don't know what why it happens. Also is a bug hard to reproduce.

> your jdk language is not english

Maybe you can try a machine completely in English :sweat_smile:

**Comment by Yensan** [comment]
@faustinoaq 
Hi, faustinoaq , I have encountered this issue too. After days of searching in github.com |  visualstudio.com | stackoverflow.com  , and  now  ...... I hate this issue ......
If you would, please give a intelij OR eclipse simple HelloWorld sample, which have project structure and vs code settings(settings.json  tasks.json   launch.json ),  thank you very much

**Comment by faustinoaq** [comment]
I recommend you Intellij or Eclipse because they are more friendly and work out the box. :sweat_smile: 

VSCode is a bit messy, because you need to figureout about bugs, misconfigurations, enviroments, etc. So I'm able to debug using this plugin, but it sometimes doesn't fit my requirements.

VSCode is better for JavaScript/Typescript, and other languages without an IDE. However Java is widely supported by almost every good IDE (Intellijj, Eclipse, Netbeans, etc).

**Comment by Yensan** [comment]
@faustinoaq 
Yes, you are right. I just want to edit all of languages in one place, and open too much window is so complex. But finally, Intelij or eclipse is better. Also vs code can edit python, but when I want to develop program nor script, I will use pyCharm. (what I said just looks like a jet brain's Ad,  ^_^ )

**Comment by LimnN** [comment]
@faustinoaq I have tried restart or reload for several times, but nothing changed

**Comment by LimnN** [comment]
And I found an error in the output window:
![image](https://user-images.githubusercontent.com/14120609/30841115-53c18a36-a2ad-11e7-8117-003d7a8734c1.png)

**Comment by faustinoaq** [comment]
Currenlty I'm able to debug using the latest version of this extension:

![screenshot_20170925_222832](https://user-images.githubusercontent.com/3067335/30841204-ddd1de6e-a240-11e7-9b4d-e18adcd61f67.png)

I don't know what would be happening with you configuration :sweat_smile:

**Comment by faustinoaq** [comment]
I'm using the default configuration

**Comment by LimnN** [comment]
@faustinoaq I set the launch.json same as yours, but still failed. the stepin stepout button I never have seen them become clickable. and the first green button, it always grey or a blue pause icon.

**Comment by faustinoaq** [comment]
@LimnN I don't know what would be happening, but If VSCode don't work for you, try Intellij Idea or Eclipse because they have better support for java debugging out-the-box :sweat_smile:
